jobspec/parse.go

```diff
@@ -1042,7 +1043,7 @@ func parseChecks(service *api.Service, checkObjs *ast.ObjectList) error {
 		if ot, ok := co.Val.(*ast.ObjectType); ok {
 			checkRestartList = ot.List
 		} else {
-			return fmt.Errorf("check_restart '%s': should be an object", check.Name)
+			return fmt.Errorf("check '%s': should be an object", check.Name)
```

Comment:
Not sure I get why you changed this? It is the check restart object that is failing the cast.

Reply:
Argh, I think the `check.Name` interpolation confused me. Fixing.